modin/logging/config.py

```diff
@@ -96,7 +97,12 @@ def configure_logging():
 
     os.makedirs(os.path.dirname(log_filename), exist_ok=True)
 
-    logfile = logging.FileHandler(log_filename, "a")
+    logfile = RotatingFileHandler(
+        filename=log_filename,
+        mode="a",
+        maxBytes=100000000,
```

Comment:
Why do we have `maxBytes` set so high?

Reply:
Where is the 10GB max?

Reply:
Sounds good to me!

Reply:
This looks like 100 MB (10^8)? Is that what you meant to put? I think something like `int(1e8)` would be more readable.

Reply:
I think it might be better to make this configurable instead of hard coded. 

I think setting a default max of 10GB of logs is way too high. This is 10GB per Modin job, not total. I prefer to cap it at 100MB per job by default. If more is needed, they can set a configuration.

Reply:
So 10MB per file, and a max of 10 files, for a total of 100MB per job?